src/beliefpropagation/beliefpropagation.jl

```diff
@@ -81,17 +81,25 @@ function belief_propagation_iteration(
   mts::DataGraph;
   contract_kwargs=(; alg="density_matrix", output_structure=path_graph_structure, maxdim=1),
   compute_norm=false,
+  update_order::String="Parallel",
+  es=edges(mts),
```

Comment:
I would call it `edges`, `es` is a bit cryptic. You can use `Graphs.edges` to refer to the function.